hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/ExponentialRetryPolicy.java

```diff
@@ -31,19 +34,19 @@ public class ExponentialRetryPolicy extends AbfsRetryPolicy {
    * Represents the default amount of time used when calculating a random delta in the exponential
    * delay between retries.
    */
-  private static final int DEFAULT_CLIENT_BACKOFF = 1000 * 3;
+  private static final int DEFAULT_CLIENT_BACKOFF = DEFAULT_BACKOFF_INTERVAL;
```

Comment:
These variable names are also redundant right?
We can simply replace their usage in constructor with the imported once.

Reply:
Taken